src/inferenceresponse.hpp

```diff
@@ -52,7 +52,7 @@ class InferenceResponse {
     Status setId();
     Status getId();
     InferenceParameter* getInferenceParameter(const char* name);
-    // TODO this can be removed with prediction tests overhaul
+
     void Clear();
```

Comment:
```suggestion
    void clear();
```

Reply:
It needs to be consistent with TFS/KFS due to templatization